src/ui/button/BaseButton/BaseButton.scss

```
@@ -0,0 +1,10 @@
+@use "../../../styles/globals.scss" as *;
```

```suggestion
@use "../../../styles/utils/index.scss" as *;
```

Глобальные стили подключаются в приложение. А для компонентов используются утилиты